README.md

```diff
@@ -1,4 +1,4 @@
-virtual-machine
+create virtual-machine network gateway
```

Comment:
Be a sentence, Upper the first character?

Reply:
sure